src/easee/client.go

```diff
@@ -252,9 +257,11 @@ func (c *Client) newRequest(method, path string, body interface{}) (*http.Reques
 func (c *Client) do(req *http.Request, v interface{}) (*http.Response, error) {
 	resp, err := c.httpClient.Do(req)
 	if err != nil {
+
```

Comment:
Empty line is needed only when other expressions are in the scope. Here it should just stay without.